rosbag2_cpp/test/rosbag2_cpp/test_sequential_writer.cpp

```diff
@@ -579,7 +580,7 @@ TEST_F(TemporaryDirectoryFixture, split_bag_metadata_has_full_duration) {
   };
   rosbag2_storage::StorageOptions storage_options;
   storage_options.uri = (rcpputils::fs::path(temporary_dir_path_) / "split_duration_bag").string();
-  storage_options.storage_id = "sqlite3";
+  storage_options.storage_id = rosbag2_storage::get_default_storage_id();
```

Comment:
Would it make sense to leave this as hardcoded, with the aim to parametrize this test across both `("sqlite3", "mcap")` when both are installed in the CI environment?